psi4/src/export_efp.cc

```diff
@@ -39,8 +39,10 @@ void export_efp(py::module& m)
     // "no_init" and the constructor definition, def(init<Options&>())
     py::class_<EFP, std::shared_ptr<EFP> >(m, "EFP", "Class interfacing with libefp").
         def(py::init<Options&>()).
+#ifdef USING_libefp
         def("compute", &EFP::compute, "Computes libefp energies and, if active, torque").
         def("set_qm_atoms", &EFP::set_qm_atoms, "Provides libefp with QM fragment information").
-        def("nfragments", &EFP::get_frag_count, "Returns the number of EFP fragments in the molecule").
-        def("print_out", &EFP::print_out, "Prints options settings and EFP and QM geometries");
+        def("print_out", &EFP::print_out, "Prints options settings and EFP and QM geometries").
+#endif 
+        def("nfragments", &EFP::get_frag_count, "Returns the number of EFP fragments in the molecule");
```

Comment:
My main concern was the auto-doc stuff, but that's apparently a non-issue.  It'd be nice for the pydocs to be consistent with user compilation settings, but certainly not a big deal.